generated/python/gapic-google-cloud-speech-v1beta1/google/cloud/gapic/speech/v1beta1/speech_api.py

```diff
@@ -0,0 +1,218 @@
+
+        Args:
+          service_path (string): The domain name of the API remote host.
+          port (int): The port on which to connect to the remote host.
+          channel (:class:`grpc.Channel`): A ``Channel`` instance through
+            which to make calls.
+          ssl_creds (:class:`grpc.ChannelCredentials`): A
+            ``ChannelCredentials`` instance for use with an SSL-enabled
+            channel.
+          client_config (dict):
+            A dictionary for call options for each method. See
+            :func:`google.gax.construct_settings` for the structure of
+            this data. Falls back to the default config if not specified
+            or the specified config is missing data points.
+          metadata_transformer (Callable[[], list]): A function that creates
+             the metadata for requests.
+          app_name (string): The codename of the calling service.
+          app_version (string): The version of the calling service.
+
+        Returns:
+          A SpeechApi object.
+        """
+        if scopes is None:
+            scopes = self._ALL_SCOPES
+        if client_config is None:
+            client_config = {}
+        goog_api_client = '{}/{} {} gax/{} python/{}'.format(
+            app_name, app_version, self._CODE_GEN_NAME_VERSION,
+            self._GAX_VERSION, platform.python_version())
+        metadata = [('x-goog-api-client', goog_api_client)]
+        default_client_config = json.loads(
+            pkg_resources.resource_string(
+                __name__, 'speech_client_config.json').decode())
+        defaults = api_callable.construct_settings(
+            'google.cloud.speech.v1beta1.Speech',
+            default_client_config,
+            client_config,
+            config.STATUS_CODE_NAMES,
+            kwargs={'metadata': metadata})
+        self.speech_stub = config.create_stub(
+            cloud_speech_pb2.SpeechStub,
+            service_path,
+            port,
+            ssl_creds=ssl_creds,
+            channel=channel,
+            metadata_transformer=metadata_transformer,
+            scopes=scopes)
+
+        self._sync_recognize = api_callable.create_api_call(
+            self.speech_stub.SyncRecognize,
+            settings=defaults['sync_recognize'])
+        self._async_recognize = api_callable.create_api_call(
+            self.speech_stub.AsyncRecognize,
+            settings=defaults['async_recognize'])
+        self._streaming_recognize = api_callable.create_api_call(
+            self.speech_stub.StreamingRecognize,
+            settings=defaults['streaming_recognize'])
+
+    # Service calls
+    def sync_recognize(self, config, audio, options=None):
+        """
+        Perform synchronous speech-recognition: receive results after all audio
+        has been sent and processed.
+
+        Example:
+          >>> from google.cloud.gapic.speech.v1beta1 import speech_api
+          >>> from google.cloud.speech.v1beta1 import cloud_speech_pb2
```

Comment:
Need to update path. Use the pattern `s/google\.cloud\(\.[g].*_pb2\)/google.cloud.grpc\1/`.